src/components/lib/FlowGrid/FlowGrid.jsx

```diff
@@ -209,15 +209,26 @@ export default class FlowGrid extends Component{
     )
   }
 
+  componentDidMount() {
+    if (__DEV__) { window.RecordMountEvent(this) }
```

Comment:
This isn't super Dry.  Maybe in a module is on the top level, then we call that from places like this.

`` import {recorder} ``
``recorder.RecordMountEvent(this)`` (the IF DEV) happens in that module, not everywhere else)